Simplenote/Classes/UIView+Animations.swift

```diff
@@ -142,9 +142,10 @@ extension UIView {
         let originalTransform = transform
 
         transform = leftTranslation
-        UIView.animate(withDuration: 0.07, delay: 0.0, options: [.autoreverse, .repeat]) {
-            UIView.setAnimationRepeatCount(5)
-            self.transform = rightTranslation
+        UIView.animate(withDuration: 0.07, delay: 0.0, options: []) {
```

Comment:
Should we replace these numbers with some sort of constant?  Replace the `0.0` s with `.zero` and either add some local constants to the file for the others, or add them to `UIKitConstants`

Reply:
We should *always* avoid magic numbers (inline constants).However, we should probably cap the kind of changes we're introducing in this PR (which is already super large!!)